### PR TITLE
Calculate fee for burnr transactions

### PR DESCRIPTION
**Pull request by wirednkod**
@goldsteinsveta any ideas on how we present nicely this one?

### CODE REVIEW

**Comment by wirednkod** [comment]
Fixes #182

**Comment by raoulmillais** [comment]
> Fixes #182

You have to add this to the first comment in the PR or it won't automatically close

**Comment by raoulmillais** [comment]
I don't think I understand burnr well enough to review this properly so I'll defer to @goldsteinsveta

**Comment by goldsteinsveta** [comment]
uh, this is weird:

Screenshot 2021-04-30 at 18 19 20

**Review by goldsteinsveta — APPROVED** [review]
I guess fee calculation is correct. But I was not able to test transfers on Westend. The formatting of balance values is incorrect and I feel like it should be simplified and better systematised. I don't see a big problem with merging this, as the problem is already there in the deployed version. But this should be the next TODO on burnr

File: projects/burnr/src/components/InputFunds.tsx [hunk 1]
```diff
@@ -12,16 +12,20 @@ interface Props {
...
-	};
-	const handleChange = (e: ChangeEvent) => {
-    const value = (e.currentTarget as HTMLButtonElement).value;
-    const v: number = parseFloat(value);
-		setShowValue(value !== '' ? value : '');
-		setAmount(value !== '' ? (v * 1000000000000).toString() : '0');
+
+	const handleChange = (e: ChangeEvent | MouseEvent, fromButtons = false) => {
+    if (fromButtons) {
+      const calcNewTotal = (parseFloat((e.currentTarget as HTMLButtonElement).value) * parseInt(new BN(total).toString()))/1000000000000;
```
[the remark this answers is not in the document]
Comment (wirednkod) [reply projects/burnr/src/components/InputFunds.tsx]: Yes and no - Yes I should use something from api (and I did) - no its not exactly the one you pointed but close. Thank you for pointing that out 👍🏽

**Comment by wirednkod** [comment]
> I guess fee calculation is correct. But I was not able to test transfers on Westend. The formatting of balance values is incorrect and I feel like it should be simplified and better systematised. I don't see a big problem with merging this, as the problem is already there in the deployed version. But this should be the next TODO on burnr

The issue that was making burnr wallet to brake is fixed now and the code merged from main changes you last did and pushed @goldsteinsveta

**Comment by goldsteinsveta** [comment]
Fee calculation is correct.

Some problems that could be addressed later:
- formatting of values (unit prefixes) is still tricky, it's fine to only use base unit is fine for such a simplistic UI.
- how much "the receiver will get" will always be the same value as input field. The fee does not affect sent value, it is deduced from the sender's account in addition to sent value. I think we should show "sender's balance after sent tx" in place of "the receiver will get" value

**Review by goldsteinsveta — APPROVED** [review]

**Pull request merged**